agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3110,6 +3132,30 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             break;
         }
```

Comment:
This check should not be part of STATE_GET_SUPPORTED_CHANNELS. It is there in STATE_JOIN_MASTER because we do send_cmdu_to_controller there. In this state, we don't do anything like that, so it's not needed.

Reply:
Done.